package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mock-req-res",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
please don't update the version number.  I will do that when I release a new package.  As this PR only affects development dependencies there will not be a need to release a new package.